Doc/library/sched.rst

```diff
@@ -131,8 +130,10 @@ Scheduler Objects
    .. versionchanged:: 3.3
       *blocking* parameter was added.
 
+Instance attributes (read-only):
```

Comment:
I prefer to keep the "read-only" mention on the attribute doc.

Reply:
Both changes made/reverted.